packages/waku/tests/create-pages.test.ts

```diff
@@ -685,7 +685,7 @@ describe('createPages', () => {
       {
         elements: {
           root: { isStatic: true },
-          'page:/test/w/x': { isStatic: true },
+          'page:/test/[a]/[b]': { isStatic: true },
```

Comment:
I think it's generally a good idea for new_createPages and I believe the current createPages behaves like this.

One note is that this string is a slotId, which is a cache id.
So, technically, the following two are very different behavior-wise:
- Having `/src/pages/foo.tsx` and `/src/pages/bar.tsx`,
- Having `/src/pages/[slug].tsx` with staticPath = `['foo', 'bar']`.

Reply:
If I follow you correctly, We'd have better ability to more granularly cache the page slots if this is: `'page:/test/w/x'`?

Will `defineRouter` handle `'page:/test/[a]/[b]'` and `'page:/test/w/x'` the same right now?

Reply:
Yeah, it handles the same. `'page:...'` is the createPages thing. defineRouter only knows it's a string id.

Reply:
> I believe the current createPages behaves like this.

On second thought, that doesn't hold for the new one. It's very differently designed. Not just refactoring.